katalog/configs/audit/kustomization.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2022 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  # - audit-hosttrailer.yml
```

Comment:
if this is commented out because is not needed, I think it would be better to delete it.
If this is optional let's add a comment explaining when to uncomment

Reply:
It is commented out because the feature i needed was not implemented. I will upgrade the logging operator to the latest version and update the audit package accordingly

Reply:
Solved updating the operator to 3.17.2